src/lib/features/project-status/projects-status.e2e.test.ts

```diff
@@ -256,3 +256,33 @@ test('project health should be correct average', async () => {
 
     expect(body.averageHealth).toBe(40);
 });
+
+test('project status contains lifecycle data', async () => {
```

Comment:
Could we have a small example also that data calculations work. Meaning, adding some data to database that will be returned.

Reply:
sure, I can add that in a follow-up, but that is essentially what all the tests in `src/lib/features/project-status/project-lifecycle-summary-read-model.test.ts` are there for.